install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Doesnt work with **MacOS Big Sur**

Reply:
do you have bash installed?

Reply:
bruh yes its just that I picked a random line for the review

Reply:
what is the error you get

Reply:
> what is the error you get

See the last comment